sdcv.el

```diff
@@ -596,7 +608,7 @@ Argument DICTIONARY-LIST the word that need transform."
     (dolist (word words)
```

Comment:
You are absolutely right.  But after I look throught the script, it seems only this `dolist` actually uses cl-lib, maybe it should be
```diff
-           (return word)
+           word
```
and rid of the cl-lib dependency all together ?

Reply:
you can use throw/catch instead of cl-return:
```elisp
(catch 'result
  (dolist (word words)
    (if ... (throw 'result word) ...)))
```

Reply:
> you can use throw/catch instead of cl-return:
> 
> ```emacs-lisp
> (catch 'result
>   (dolist (word words)
>     (if ... (throw 'result word) ...)))
> ```

Done #24 , I think this PR can be close now, but free to comment or re-open if necessary.